webui/ocw/lib/db.py

```diff
@@ -178,6 +178,7 @@ def __update_run():
                 instances = [azure_to_local_instance(i, vault_namespace) for i in instances]
                 logger.info("Got %d resources groups from Azure", len(instances))
                 sync_csp_to_local_db(instances, ProviderChoice.AZURE, vault_namespace)
+                Azure(vault_namespace).cleanup_all()
```

Comment:
I would prefer to keep it in separate function

Reply:
I thought about it as well. But then I need to implement that loop over CSP's and namespaces again.

Reply:
and I think it is right way to go ...

Reply:
done